opencensus/plugins/grpc/internal/measures.cc

```diff
@@ -33,117 +33,87 @@ constexpr char kCount[] = "1";
 }  // namespace
 
 // Client
-stats::MeasureInt64 RpcClientErrorCount() {
-  static stats::MeasureInt64 measure = stats::MeasureInt64::Register(
-      kRpcClientErrorCountMeasureName, "RPC Errors", kCount);
+stats::MeasureDouble RpcClientSentBytesPerRpc() {
+  static const auto measure = stats::MeasureDouble::Register(
+      kRpcClientSentBytesPerRpcMeasureName,
+      "Total bytes sent across all request messages per RPC", kUnitBytes);
   return measure;
 }
 
-stats::MeasureDouble RpcClientRequestBytes() {
-  static stats::MeasureDouble measure = stats::MeasureDouble::Register(
-      kRpcClientRequestBytesMeasureName, "Request bytes", kUnitBytes);
-  return measure;
-}
-
-stats::MeasureDouble RpcClientResponseBytes() {
-  static stats::MeasureDouble measure = stats::MeasureDouble::Register(
-      kRpcClientResponseBytesMeasureName, "Response bytes", kUnitBytes);
+stats::MeasureDouble RpcClientReceivedBytesPerRpc() {
+  static const auto measure = stats::MeasureDouble::Register(
+      kRpcClientReceivedBytesPerRpcMeasureName,
+      "Total bytes received across all response messages per RPC", kUnitBytes);
   return measure;
 }
 
 stats::MeasureDouble RpcClientRoundtripLatency() {
-  static stats::MeasureDouble measure = stats::MeasureDouble::Register(
-      kRpcClientRoundtripLatencyMeasureName, "RPC roundtrip latency msec",
+  static const auto measure = stats::MeasureDouble::Register(
+      kRpcClientRoundtripLatencyMeasureName,
+      "Time between first byte of request sent to last byte of response "
+      "received, or terminal error.",
```

Comment:
This is the only description ending in a full stop. Suggest remove it for consistency.

Reply:
Done.